packages/blockchain-wallet-v4-frontend/src/data/modals/types.ts

```diff
@@ -160,6 +160,7 @@ export type ModalOriginType =
   | 'WalletBalanceDropdown'
   | 'WalletConnect'
   | 'WelcomeModal'
+  | 'Withdrawal'
   | 'WithdrawModal'
```

Comment:
arent these the same thing?
 | 'Withdrawal'
  | 'WithdrawModal'

Reply:
right we can use WithdrawModal, let me fix